test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/HistoryTests.cs

```diff
@@ -75,9 +75,11 @@ public async Task GivenAValueForSince_WhenGettingSystemHistory_TheServerShouldRe
             Thread.Sleep(500);  // put a small gap between since and the first edits
             _createdResource.Resource.Text = new Narrative { Div = "<div>Changed by E2E test</div>" };
 
-            var updatedResource = await _client.UpdateAsync<Observation>(_createdResource);
+            var tag = new Coding(string.Empty, Guid.NewGuid().ToString());
```

Comment:
probably we can create just guid here(var tag = Guid.NewGuid().ToString()) and move 'new Coding(string.Empty, tag);' into the method AddTagToResource()

Reply:
Updated.